vm-virtio/src/vhost_user/handler.rs

```diff
@@ -0,0 +1,259 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
```

Comment:
Why is that BSD-3-Clause licensed? Does that contain some crosvm code? If so we should keep their copyright as well.

Reply:
I think this comes from the Firecracker code, which includes the following copyrights:

```
// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
//
// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the THIRD-PARTY file.
```

We should keep it, change the SPDX identifier and add our copyright.

Reply:
Yes, handler.rs is originally from firecracker, added their copyright.